SPECS/audit/audit.spec

```diff
@@ -12,13 +12,14 @@ Patch0:         refuse-manual-stop.patch
 BuildRequires:  e2fsprogs-devel
 BuildRequires:  krb5-devel
 BuildRequires:  swig
-BuildRequires:  systemd-bootstrap
```

Comment:
systemd-bootstrap is not needed to build audit. removed here and in toolchain script below.